weave-js/src/components/PagePanelComponents/Home/Browse3/pages/PlaygroundPage/PlaygroundChat/PlaygroundCallStats.tsx

```diff
@@ -43,21 +31,34 @@ export const PlaygroundCallStats = ({call}: {call: TraceCallSchema}) => {
     false
   );
 
+  const latency = call?.summary?.weave?.latency_ms ?? 0;
+  const usageData = call?.summary?.usage;
+  const costData = call?.summary?.weave?.costs;
+
   return (
     <Tailwind>
-      <div className="flex w-full flex-wrap items-center justify-center gap-8 py-8 text-sm text-moon-500">
-        <span>Latency: {latency}ms</span>
-        <span>•</span>
+      <div className="flex w-full items-center justify-center gap-8 py-8">
+        <TraceCostStats
+          usageData={usageData}
+          costData={costData}
+          latency_ms={latency}
+          costLoading={false}
+        />
         {(call.output as any)?.choices?.[0]?.finish_reason && (
-          <>
-            <span>
-              Finish reason: {(call.output as any).choices[0].finish_reason}
-            </span>
-            <span>•</span>
-          </>
+          <Tooltip
+            content="Finish Reason"
+            trigger={
+              <span>
```

Comment:
I think a Pill renders a div, so this should probably be a block level element like div too.